.github/workflows/on-demand-build-docker-image-deploy-preview.yml

```
@@ -5,52 +5,79 @@ on:
+            echo "pr_number=${{ github.event.client_payload.pull_request.number }}" >> "$GITHUB_OUTPUT"
+          else
+            echo "pr_number=${{ inputs.issue_number }}" >> "$GITHUB_OUTPUT"
+          fi
+
   notify-job-details:
+    needs: [resolve-params]
     runs-on: ubuntu-latest
     steps:
       # This step creates a comment on the PR with a link to this workflow run.
       - name: Add a comment on the PR with link to workflow run
         uses: peter-evans/create-or-update-comment@v3
```

actionlint (1.7.11)

[error] the runner of "peter-evans/create-or-update-comment@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)